code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -61,6 +65,11 @@
 	remove_self(removed)
 	return
 
+
+//Initial effect is called once when the reagent first starts affecting a mob.
+/datum/reagent/proc/initial_effect(var/mob/living/carbon/M, var/alien)

```

Comment:
Isn't referenced anywhere else, but in changes to this file.

Null change, so please remove.